features/support/helpers.rb

```diff
@@ -11,12 +11,38 @@
   PaaS: 'Platform as a Service',
   IaaS: 'Infrastructure as a Service',
   SCS: 'Specialist Cloud Services'
-}
+}.freeze
+
+META_DATA_XPATHS = {
+  location: '//*[@class="search-result"]//*[@class="search-result-important-metadata"][1]//*[@class="search-result-metadata-item"][2]',
+  lot: '//*[@class="search-result"]//*[@class="search-result-metadata"][1]//*[@class="search-result-metadata-item"][1]',
+  role: '//*[@class="search-result"]//*[@class="search-result-metadata"][1]//*[@class="search-result-metadata-item"][2]',
+  status: '//*[@class="search-result"]//*[@class="search-result-metadata"][2]//*[@class="search-result-metadata-item"][1]'
+}.freeze
+
+CLOSED_OUTCOMES = {
+  Awarded: 'Closed: awarded',
+  Cancelled: 'Closed: cancelled',
+  Closed: 'Closed: awaiting outcome',
+  Unsuccessful: 'Closed: no suitable suppliers'
+}.freeze
 
 def full_lot(lot)
   LOTS[lot.to_sym]
 end
 
+def search_result_metadata_items(type)
+  all(:xpath, META_DATA_XPATHS[type])
+end
+
+def closed_outcome?(status)
+  CLOSED_OUTCOMES.key?(status.to_sym)
```

Comment:
I am going to have to properly read up on Ruby symbols at some point, aren't I... 😸

Reply:
Yeah, the only reason I did that was because there was the `LOTS` dictionary making use of symbols as keys. Otherwise a string key hash would have sufficed without the need to convert the passed in key to symbol:

```
hash = {
  'foo' => 'bar
}
```

Ruby loves to give us option! 😃